search/includes/classes/class-search.php

```diff
@@ -2174,9 +2174,11 @@ public function limit_max_result_window( $current_value ) {
 	}
 
 	/**
-	 * Determine whether the rate-limiting is in effect
+	 * Determine whether the search rate-limiting is in effect
+	 *
+	 * @return bool If search rate limiting is in effect
 	 */
-	public static function is_rate_limited(): bool {
+	public static function is_search_rate_limited(): bool {
```

Comment:
https://github.com/search?q=org%3Awpcomvip%20is_rate_limited&type=code :-(

We may want to add a stub for `is_rate_limited()` with `_doing_it_wrong()` or a deprecation warning.

Reply:
Ah, good check. I changed it back and added a comment to the PHPDoc with clarification for the path of least resistance :|